components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa/src/Publisher/Models/EngineConfigurationModel.cs

```diff
@@ -30,5 +30,12 @@ public class EngineConfigurationModel {
         /// Diagnostics setting
         /// </summary>
         public TimeSpan? DiagnosticsInterval { get; set; }
+
+        /// <summary>
+        /// Define the maximum number of messages, 
```

Comment:
Same here, please mention it being maximum number of messages in a buffer.

Reply:
updated XML comment